sdk/Trace/Span.php

```diff
@@ -74,28 +73,28 @@ public function setSpanStatus(int $code, ?string $description = null): API\Span
         return $this;
     }
 
-    public function end(string $timestamp = null): API\Span
+    public function end(int $timestamp = null): API\Span
```

Comment:
I think we need to come up with a name for what we call the result of an `hrtime` -- `$timestamp` is misleading, and at least as written as this uses a duration (which makes sense, but we may want both).

Reply:
Agreed; I think we need to have some better naming scheme.  I saw you asked in #open-telemetry, let's see what they say.